Makefile

```diff
@@ -0,0 +1,44 @@
+SHELL := /bin/sh
+PATH := bin:$(PATH)
+
+#####################
+# Docker/Containers #
+#####################
+build:
+	uid=$$(id -u) gid=$$(id -g) docker-compose build
+
+up:
+	docker-compose up
+
+clean:
+	docker-compose rm -vsf
+	docker-compose down -v --remove-orphans
+	sudo rm -f demo/docroot/sites/default/settings.php
```

Comment:
Thank you this Makefile is a great improvement.

May I suggest a minor tweak from field testing ... I had a broken build in which a yarn-error.log was generated.
My expectation is that a "make clean" should clean this up... otherwise if the next developer goes through a cycle of make build .. see error 1... fix error 1 ... make build .. move onto error2  see the stale yarn-error.log from error1 and get misdirected about the nature of error2.

It is a bit contrived hey I have already had that one bite me in the behind.  

so in short 

rm yarn-error.log